t/016-resp-header.t

```diff
@@ -1976,14 +1976,12 @@ Content-Type: application/json
     }
 --- request
 GET /t
---- error_code: 500
 --- response_headers
-header:
+header: value%0Dfoo:bar%0Abar:foo
```

Comment:
Do we need to update the documentation to reflect the behavioral changes in this PR?